packages/bridge-ui-v2/src/components/Bridge/ProcessingFee/ProcessingFee.svelte

```diff
@@ -157,7 +157,7 @@
 
   <dialog id={dialogId} class="modal" class:modal-open={modalOpen}>
     <div class="modal-box relative px-6 py-[35px] md:rounded-[20px] bg-neutral-background">
-      <button class="absolute right-6 top-[35px]" on:click={cancelModal}>
```

Comment:
Looking at all the close buttons, they are "almost" identical, when they really should be "completely" identical. Let's deploy this fix, but I think we should have a separate Dialog Component as a future improvement